integration-tests/src/test/resources/onpremdomain/config/config.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+  Copyright (c) 2021, Oracle and/or its affiliates.
+  Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+-->
+
+<domain xmlns="http://xmlns.oracle.com/weblogic/domain" xmlns:sec="http://xmlns.oracle.com/weblogic/security" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:wls="http://xmlns.oracle.com/weblogic/security/wls" xsi:schemaLocation="http://xmlns.oracle.com/weblogic/security/wls http://xmlns.oracle.com/weblogic/security/wls/1.0/wls.xsd http://xmlns.oracle.com/weblogic/domain http://xmlns.oracle.com/weblogic/1.0/domain.xsd http://xmlns.oracle.com/weblogic/security/xacml http://xmlns.oracle.com/weblogic/security/xacml/1.0/xacml.xsd http://xmlns.oracle.com/weblogic/security/providers/passwordvalidator http://xmlns.oracle.com/weblogic/security/providers/passwordvalidator/1.0/passwordvalidator.xsd http://xmlns.oracle.com/weblogic/security http://xmlns.oracle.com/weblogic/1.0/security.xsd">
+  <name>onprem_domain</name>
+  <domain-version>12.2.1.4.0</domain-version>
```

Comment:
Do you want to hardcode it? later on when we start progressing with wls versions, is it going to be a trouble?

Reply:
I did not hard code the version. I just added the config.xml from an on-prem domain. I checked with Richard and I don't think I should remove it. Since our default weblogic version is 12.2.1.4, it works fine. I tried with config file from 12.1.3, 12.2.15 and 14.1.2 manually and it seems to work fine. It may not work if the config file has any features added in 14.1.2, since the relevant MBeans may not be available in our default weblogic image. So, in short, I think we should keep the version here, so we will know that it came from a 12.2.14 domain. We can maybe add test for other versions.